add script to init first admin (#114)
GoogleCloudPlatform/cloud-run-anthos-reference-web-app: README.md

## Try Out the Application

Once your application is deployed, you can try it out by navigating to `https://$DOMAIN`,
where `$DOMAIN` is the custom domain you configured in `env.mk`.

### Setup first admin user

After you login at least once to the app, you can use this script to update your account to `admin`.
Afterwards you can use Users page to manage other accounts. To use this script you will need to follow the

> guide of [Initialize the firebase Admin SDK][] and setup `GOOGLE_APPLICATION_CREDENTIALS` environment variable.

Reviewer: nit: wording

```suggestion
After you login at least once to the app, you can use this script to make your account an `admin`.
Afterwards you'll be able to use the Users page to manage other accounts. To use this script you will need to  [Initialize the Firebase Admin SDK][] and setup the `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
```

On a side note, it's kind of shame we need to have users go through extra steps to get this to work. I can't think of a better way though :/

## Try Out the Application

Once your application is deployed, you can try it out by navigating to `https://$DOMAIN`,
where `$DOMAIN` is the custom domain you configured in `env.mk`.

### Setup first admin user

After you login at least once to the app, you can use this script to make your
account an `admin`. Afterwards you'll be able to use the Users page to manage

> other accounts. To use this script you will need to follow the guide of

Reviewer: minor nit: "initialize" is a verb so I suggest changing this to at least
```suggestion
other accounts. To use this script you will need to follow the guide to
```
or more simply:
```suggestion
other accounts. To use this script you will need to
```

> [Initialize the Firebase Admin SDK][]and setup

Reviewer: nit: missing space

```suggestion
[Initialize the Firebase Admin SDK][] and setup
```

[domain-setup.sh]: scripts/domain-setup.sh
[firestore rules page screenshot]: docs/img/firestore_rules_page.png
[`firestore/firestore.rules`]: firestore/firestore.rules

> [Initialize the firebase Admin SDK]: https://firebase.google.com/docs/admin/setup#initialize-sdk

Reviewer: nit 

```suggestion
[Initialize the Firebase Admin SDK]: https://firebase.google.com/docs/admin/setup#initialize-sdk
```
